src/response.rs

```diff
@@ -34,7 +34,8 @@ impl WebDriverResponse {
         }.unwrap();
 
         match self {
-            WebDriverResponse::Generic(_) => obj,
+            WebDriverResponse::Generic(_) |
+            WebDriverResponse::Cookie(_) => obj,
```

Comment:
Should this not be wrapped in `{value: ["window handle 1", "window handle 2"]}`?